pyecoregen/templates/package.py.tpl

```diff
@@ -7,6 +7,9 @@ from .{{ element.name }} import name, nsURI, nsPrefix, eClass
     from .{{ element.name }} import {{ element.eClassifiers | join(', ', attribute='name') }}
 {%- endif %}
 
+{% for package, classifs in imported_classifiers_package.items() -%}
+    from {{ package | pyfqn }} import {{ classifs|map(attribute='name')|join(', ') }}
```

Comment:
Please separate the filter expressions by spaces around the bar as done in the other template code.

Reply:
Done!